pkgs/by-name/ge/gentium/package.nix

```diff
@@ -43,6 +43,7 @@ stdenvNoCC.mkDerivation rec {
     '';
     downloadPage = "https://software.sil.org/gentium/download/";
     maintainers = with maintainers; [
+      b-fein
```

Comment:
Wrong file edited. Also commit message should start with `gentium-plus:`

Reply:
Which commit do you refer to? I think 487ebcfd1eb1cb92e56933385ce77a2824a5dae9 edits the `gentium` package file and starts with `gentium: `. I intended to only add myself as maintainer for the new `gentium` and `gentium-book` packages rather than the old `gentium-plus` one.